changelog/57842.fixed

```diff
@@ -0,0 +1 @@
+Updating Slack engine to use slack_bolt library.
```

Comment:
I think the file extension name should be .changed

Reply:
or `.added` if we agree to put removal of slackclient on a deprecation path

Reply:
@Ch3LL Using `slack_client` is not longer an option, it is deprecated by Slack and cannot be used to connect to Slack anymore.